tcms/testplans/static/testplans/style/get.css

```diff
@@ -0,0 +1,5 @@
+.js-test-case-expand-text {
+    height: 300px;
+    width: 100%;
+    border: none;
+}
```

Comment:
It looks to me this entire style isn't necessary. From what I can see width is automatically stretched to fill the parent div and height should be automatic b/c some texts will be shorter, others will be longer.